### PR TITLE
1264: Update actions that have deprecations

### PR DESCRIPTION
**Pull request by shaunharrisonFR**
Issue: https://github.com/SecureApiGateway/SecureApiGateway/issues/1264

### CODE REVIEW

File: kustomize/overlay/7.3.0/devops/configmap.yaml [hunk 1]
```diff
@@ -8,3 +8,4 @@ data:
   MTLS_FQDN: mtls.sapig.devops.forgerock.financial
   IDENTITY_PLATFORM_FQDN: iam.devops.forgerock.financial
   OB_ASPSP_SOFTWARE_ID: gYcVCmFmzItBTxM4Y1cPn2
+  IG_OB_ASPSP_SIGNING_KID: KT85P5YZolPsbGVmHIOCm-KgRT0
```
Comment (dbadham-fr) [inline kustomize/overlay/7.3.0/devops/configmap.yaml:11]: What does this have to do with the deprecated actions?

**Review by dbadham-fr — CHANGES_REQUESTED** [review]

File: kustomize/overlay/7.3.0/devops/configmap.yaml (hunk 1 above)
Comment (shaunharrisonFR) [reply kustomize/overlay/7.3.0/devops/configmap.yaml:11]: ah looks like this change was outstanding, will sort

**Review by dbadham-fr — APPROVED** [review]
